plugin/metrics/prometheus/options.go

```diff
@@ -42,24 +42,19 @@ const (
 	defaultTokenFilePath  = ""
 
 	defaultSupportSpanmetricsConnector = true
-	defaultMetricNamespace             = ""
+	defaultMetricNamespace             = "prometheus"
```

Comment:
this isn't right. You're confusing two different "namespace" notions. This one is what is used in the metrics themselves. What you're deleting is a "storage namespace" (which didn't make sense for metrics storage).

Reply:
Ohh okay. I will revert that change.